### ISSUE TITLE
Accessibility errors found in your template

### ISSUE THREAD

**Issue by wapyce** [mmistakes/skinny-bones-jekyll#107]
The user @carlsonsantana validate your site template "https://mmistakes.github.io/skinny-bones-jekyll/" and found these accessibility errors:
* Page https://mmistakes.github.io/skinny-bones-jekyll/:
  * Issue *0c72b8c0-a68e-4b70-b214-558226b89c4f*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle1.Guideline1_3.1_3_1.H42.2
    * **Message**: Heading tag found with no content. Text that is not intended as a heading should not be marked up with heading tags.
    * **Selector**: `#main > div > div:nth-child(1) > h1`
    * **Context**: `<h1></h1>`
  * Issue *c6dd63c3-ed33-43a7-8f3b-76024825cfb9*:
    * **Type**: Error
    * **Code**: WCAG2AA.Principle3.Guideline3_1.3_1_1.H57.2
    * **Message**: The html element should have a lang or xml:lang attribute which describes the language of the document.
    * **Selector**: `html`
    * **Context**: `<html><head>
    <meta charset="utf-8...</html>`

You can check these accessibility errors using [pa11y](https://github.com/pa11y/pa11y).
You can view the [full validation results in our website](https://wapyce.herokuapp.com/validation/validations/1f9403a6-e20e-45d8-ba44-cb5676b18746/).
